pymc3/tests/test_step.py

```diff
@@ -113,3 +113,31 @@ def test_binomial(self):
             Binomial('x', 10, 0.5)
             steps = assign_step_methods(model, [])
         self.assertIsInstance(steps, Metropolis)
+
+class TestSampleEstimates(unittest.TestCase):
```

Comment:
Also, just checked, and the test passes on the bad NUTS sampler -- think the loop has to be:

```
        for step_method in (NUTS, Metropolis, Slice):
            with Model():
                alpha = Normal('alpha', mu=0, sd=10)
                beta = Normal('beta', mu=0, sd=10, shape=2)
                sigma = Uniform('sigma', lower=0.0, upper=1.0)
                mu = alpha + beta[0]*X1 + beta[1]*X2
                Normal('Y_obs', mu=mu, sd=sigma, observed=Y)

                trace = sample(1000, step=step_method())
```
I'll muck around to see if I can make it fail on the bad version!